### ISSUE TITLE
Create a new LibreOffice project in my Documents\python folder

### ISSUE THREAD

**Issue by BBFlyer1** [spyder-ide/spyder#15658]
### What steps will reproduce the problem?

    • On Windows 10, running Spyder with the base python3 env, I clicked on the Projects Button, 
    • I typed in “LibreOffice” as the project name (with the “New directory “ radio button selected the first time I tried this.)
    • I selected “Existing directory” radio button (The second time as the create new project appeared to work in creating the new directory path is “E:\Users\Bob\Documents\python”)
    • and clicked the Create button

### Traceback
```python-traceback
--- Logging error ---
Traceback (most recent call last):
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 113, in get_comm
    return self.comms[comm_id]
KeyError: '8d7b09aeb8ed11ebad9f40234360dc30'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Users\Bob\anaconda3\lib\logging\__init__.py", line 1084, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "E:\Users\Bob\anaconda3\Scripts\spyder-script.py", line 10, in <module>
    sys.exit(main())
  File "E:\Users\Bob\anaconda3\lib\site-packages\spyder\app\start.py", line 214, in main
    mainwindow.main(options, args)
  File "E:\Users\Bob\anaconda3\lib\site-packages\spyder\app\mainwindow.py", line 2293, in main
    mainwindow = create_window(app, splash, options, args)
  File "E:\Users\Bob\anaconda3\lib\site-packages\spyder\app\mainwindow.py", line 2192, in create_window
    app.exec_()
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 48, in _dispatch
    getattr(self, msg_type)(msg)
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 172, in comm_msg
    comm = self.get_comm(comm_id)
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 117, in get_comm
    self.log.warning("No such comm: %s", comm_id)
Message: 'No such comm: %s'
Arguments: ('8d7b09aeb8ed11ebad9f40234360dc30',)
--- Logging error ---
Traceback (most recent call last):
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 113, in get_comm
    return self.comms[comm_id]
KeyError: '8d7b09aeb8ed11ebad9f40234360dc30'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Users\Bob\anaconda3\lib\logging\__init__.py", line 1084, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "E:\Users\Bob\anaconda3\Scripts\spyder-script.py", line 10, in <module>
    sys.exit(main())
  File "E:\Users\Bob\anaconda3\lib\site-packages\spyder\app\start.py", line 214, in main
    mainwindow.main(options, args)
  File "E:\Users\Bob\anaconda3\lib\site-packages\spyder\app\mainwindow.py", line 2293, in main
    mainwindow = create_window(app, splash, options, args)
  File "E:\Users\Bob\anaconda3\lib\site-packages\spyder\app\mainwindow.py", line 2192, in create_window
    app.exec_()
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 48, in _dispatch
    getattr(self, msg_type)(msg)
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 172, in comm_msg
    comm = self.get_comm(comm_id)
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 117, in get_comm
    self.log.warning("No such comm: %s", comm_id)
Message: 'No such comm: %s'
Arguments: ('8d7b09aeb8ed11ebad9f40234360dc30',)
--- Logging error ---
Traceback (most recent call last):
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 113, in get_comm
    return self.comms[comm_id]
KeyError: '8d7b09aeb8ed11ebad9f40234360dc30'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Users\Bob\anaconda3\lib\logging\__init__.py", line 1084, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "E:\Users\Bob\anaconda3\Scripts\spyder-script.py", line 10, in <module>
    sys.exit(main())
  File "E:\Users\Bob\anaconda3\lib\site-packages\spyder\app\start.py", line 214, in main
    mainwindow.main(options, args)
  File "E:\Users\Bob\anaconda3\lib\site-packages\spyder\app\mainwindow.py", line 2293, in main
    mainwindow = create_window(app, splash, options, args)
  File "E:\Users\Bob\anaconda3\lib\site-packages\spyder\app\mainwindow.py", line 2192, in create_window
    app.exec_()
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 48, in _dispatch
    getattr(self, msg_type)(msg)
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 172, in comm_msg
    comm = self.get_comm(comm_id)
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 117, in get_comm
    self.log.warning("No such comm: %s", comm_id)
Message: 'No such comm: %s'
Arguments: ('8d7b09aeb8ed11ebad9f40234360dc30',)
--- Logging error ---
Traceback (most recent call last):
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 113, in get_comm
    return self.comms[comm_id]
KeyError: '8d7b09aeb8ed11ebad9f40234360dc30'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Users\Bob\anaconda3\lib\logging\__init__.py", line 1084, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "E:\Users\Bob\anaconda3\Scripts\spyder-script.py", line 10, in <module>
    sys.exit(main())
  File "E:\Users\Bob\anaconda3\lib\site-packages\spyder\app\start.py", line 214, in main
    mainwindow.main(options, args)
  File "E:\Users\Bob\anaconda3\lib\site-packages\spyder\app\mainwindow.py", line 2293, in main
    mainwindow = create_window(app, splash, options, args)
  File "E:\Users\Bob\anaconda3\lib\site-packages\spyder\app\mainwindow.py", line 2192, in create_window
    app.exec_()
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 48, in _dispatch
    getattr(self, msg_type)(msg)
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 172, in comm_msg
    comm = self.get_comm(comm_id)
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 117, in get_comm
    self.log.warning("No such comm: %s", comm_id)
Message: 'No such comm: %s'
Arguments: ('8d7b09aeb8ed11ebad9f40234360dc30',)
--- Logging error ---
Traceback (most recent call last):
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 113, in get_comm
    return self.comms[comm_id]
KeyError: '8d7b09aeb8ed11ebad9f40234360dc30'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Users\Bob\anaconda3\lib\logging\__init__.py", line 1084, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "E:\Users\Bob\anaconda3\Scripts\spyder-script.py", line 10, in <module>
    sys.exit(main())
  File "E:\Users\Bob\anaconda3\lib\site-packages\spyder\app\start.py", line 214, in main
    mainwindow.main(options, args)
  File "E:\Users\Bob\anaconda3\lib\site-packages\spyder\app\mainwindow.py", line 2293, in main
    mainwindow = create_window(app, splash, options, args)
  File "E:\Users\Bob\anaconda3\lib\site-packages\spyder\app\mainwindow.py", line 2192, in create_window
    app.exec_()
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 48, in _dispatch
    getattr(self, msg_type)(msg)
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 172, in comm_msg
    comm = self.get_comm(comm_id)
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 117, in get_comm
    self.log.warning("No such comm: %s", comm_id)
Message: 'No such comm: %s'
Arguments: ('8d7b09aeb8ed11ebad9f40234360dc30',)
--- Logging error ---
Traceback (most recent call last):
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 113, in get_comm
    return self.comms[comm_id]
KeyError: '8d7b09aeb8ed11ebad9f40234360dc30'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Users\Bob\anaconda3\lib\logging\__init__.py", line 1084, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "E:\Users\Bob\anaconda3\Scripts\spyder-script.py", line 10, in <module>
    sys.exit(main())
  File "E:\Users\Bob\anaconda3\lib\site-packages\spyder\app\start.py", line 214, in main
    mainwindow.main(options, args)
  File "E:\Users\Bob\anaconda3\lib\site-packages\spyder\app\mainwindow.py", line 2293, in main
    mainwindow = create_window(app, splash, options, args)
  File "E:\Users\Bob\anaconda3\lib\site-packages\spyder\app\mainwindow.py", line 2192, in create_window
    app.exec_()
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 48, in _dispatch
    getattr(self, msg_type)(msg)
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 172, in comm_msg
    comm = self.get_comm(comm_id)
  File "E:\Users\Bob\anaconda3\lib\site-packages\qtconsole\comms.py", line 117, in get_comm
    self.log.warning("No such comm: %s", comm_id)
Message: 'No such comm: %s'
Arguments: ('8d7b09aeb8ed11ebad9f40234360dc30',)
```

## Versions

* Spyder version: 5.0.0 
* Python version: 3.8.8
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  1.6.0 (OK)
cookiecutter >=1.6.0          :  1.7.2 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0               :  7.22.0 (OK)
jedi =0.17.2                  :  0.17.2 (OK)
jsonschema >=3.2.0            :  3.2.0 (OK)
keyring >=17.0.0              :  22.3.0 (OK)
nbconvert >=4.0               :  6.0.7 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
paramiko >=2.4.0              :  2.7.2 (OK)
parso =0.7.0                  :  0.7.0 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.8.1 (OK)
pylint >=1.0                  :  2.7.4 (OK)
pyls >=0.36.2;<1.0.0          :  0.36.2 (OK)
pyls_black >=0.4.6            :  0.4.6 (OK)
pyls_spyder >=0.3.2           :  0.3.2 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=0.5.7             :  1.0.2 (OK)
qtconsole >=5.0.3             :  5.0.3 (OK)
qtpy >=1.5.0                  :  1.9.0 (OK)
rtree >=0.8.3                 :  0.9.7 (OK)
setuptools >=39.0.0           :  52.0.0.post20210125 (OK)
sphinx >=0.6.6                :  4.0.1 (OK)
spyder_kernels >=2.0.1;<2.1.0 :  2.0.1 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3;<2.0.0      :  1.0.2 (OK)
zmq >=17                      :  20.0.0 (OK)

# Optional:
cython >=0.21                 :  0.29.23 (OK)
matplotlib >=2.0.0            :  3.3.4 (OK)
numpy >=1.7                   :  1.20.1 (OK)
pandas >=1.1.1                :  1.2.4 (OK)
scipy >=0.17.0                :  1.6.2 (OK)
sympy >=0.7.3                 :  1.8 (OK)
```

**Comment by andfoy** [comment]
Hi @BBFlyer1, thanks for reaching out, could you please try to update your Spyder version to 5.0.3 and see if this error persists? You can use one of our Windows installers https://github.com/spyder-ide/spyder/releases/download/v5.0.3/Spyder_64bit_full.exe

**Comment by ccordoba12** [comment]
This error is fixed in our 5.0.3 version, so I'm closing this issue.

@BBFlyer1, if you don't want to use our installer, you need to wait until Anaconda updates Spyder to that version.
